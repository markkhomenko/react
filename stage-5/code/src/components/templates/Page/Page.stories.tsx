import React from "react";

import { storiesOf } from "@storybook/react";

import Page from "./Page.component";

storiesOf("Page", module).add("default", () => (
  <Page>
    <p>This is some page content</p>
  </Page>
));
