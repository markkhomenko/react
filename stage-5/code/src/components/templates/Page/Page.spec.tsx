import React from "react";
import renderer from "react-test-renderer";

import Page from "./Page.component";

test("Page renders as expected", () => {
  const component = renderer.create(
    <Page>
      <p>this is some content</p>
    </Page>
  );

  const tree = component.toJSON();

  expect(tree).toMatchSnapshot();
});
