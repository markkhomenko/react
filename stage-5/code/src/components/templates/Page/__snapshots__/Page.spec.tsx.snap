// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Page renders as expected 1`] = `
<main
  className="sc-AxjAm jFTkVl"
>
  <p>
    this is some content
  </p>
</main>
`;
